terraform/modules/govuk/security_group_rules.tf

```
@@ -506,13 +506,43 @@ resource "aws_security_group_rule" "signon_to_any_any" {
 
 resource "aws_security_group_rule" "statsd_from_apps_tcp" {
   description              = "Allow services in the App Mesh to send metrics to the mesh Statsd via TCP"
-  type                     = "egress"
+  type                     = "ingress"
   from_port                = "8125"
   to_port                  = "8125"
   protocol                 = "tcp"
   security_group_id        = module.statsd.security_group_id
   source_security_group_id = aws_security_group.mesh_ecs_service.id
 }
 
+resource "aws_security_group_rule" "mesh_service_to_any_https" {
+  description       = "Mesh services send requests to anywhere over HTTPS"
+  type              = "egress"
+  from_port         = 443
+  to_port           = 443
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = aws_security_group.mesh_ecs_service.id
+}
+
+resource "aws_security_group_rule" "mesh_service_to_any_dns" {
+  description       = "Mesh services send DNS queries anywhere"
+  type              = "egress"
+  from_port         = 53
+  to_port           = 53
+  protocol          = -1
```

I think you need to set this to:
```suggestion
  protocol          = "udp"
```

There's a warning at the top of the terraform docs about this: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule

---

Thanks, that should be fixed now.